src/components/Modals/NewCourse/NewCourseModal.vue

```diff
@@ -18,7 +19,7 @@
       :key="courseSelectorKey"
       placeholder='"CS 1110", "Multivariable Calculus", etc'
       :autoFocus="true"
-      @on-escape="closeCurrentModal"
+      @on-escape="escape"
```

Comment:
escape seems to do the same thing as closeCurrentModal used to do, but not @modal-closed has that same code that is inside that function. Should it instead call escape?